coverage.xml

```diff
@@ -0,0 +1,14159 @@
+<?xml version="1.0" ?>
```

Comment:
Should this file be in `.gitignore`?

Reply:
Actually codecov after running tests, saves a file in .xml version. And that file is analysed and pushed over the codecov.
```text
pytest --cov=./ --cov-report=xml
```
This is the line responsible for that

Reply:
Then remove this file and add it to `.gitignore`.  The codecov will make a new one after running tests, submit it, and we can ignore it.

Reply:
Okay!

Reply:
Done!